arch/arm/src/kinetis/kinetis_enet.c

```diff
@@ -872,7 +872,7 @@ static void kinetis_interrupt_work(void *arg)
     {
       /* An error has occurred, update statistics */
 
-      nerr("pending %0" PRIx32 "d ints %0lxd\n", pending, priv->ints);
+      nerr("pending %0" PRIx32 "d ints %0" PRIx32 "\n", pending, priv->ints);
```

Comment:
Why not
```suggestion
      nerr("pending %0" PRIx32 "d ints %0" PRIx32 "d\n", pending, priv->ints);
```
?
And even more: why `d` is needed in `%0" PRIx32 "d`?

Reply:
it's typo error from:
https://github.com/apache/incubator-nuttx/commit/9d02bbcfdde8636436ac9341a3a8a6018781281e